sympy/solvers/ode/tests/test_single.py

```diff
@@ -18,6 +19,7 @@
 
 
 x = Symbol('x')
+y = Symbol('y', real=True)
```

Comment:
Is the `real=True` needed here? If so what is it needed for?

Reply:
I don't think it is needed but in `test_ode.py` it was written like this so that's why I didn't change.

Reply:
Let's just make it `Symbol('y')`.

Reply:
should I squash my next commit or is it fine to commit directly?

Reply:
It's fine to commit directly. Squashing should generally be avoided. The reason I sometimes suggest to do it is either because

1) some of the commits are broken
2) changes in earlier commits are undone in later commits
3) There are many small commits with uninformative commit messages

As long as the commits build on each other and have good commit messages there is no reason to squash.

Reply:
sorry, I just squashed because I thought the commits which are used for very minor changes should not be separated.
Thanks for explaining.

Reply:
I think it's fine either way here.